### ISSUE TITLE
"Send" button added to window headers not functioning

### ISSUE THREAD

**Issue by simondw2000** [Dorako/pf2e-dorako-ux#15]
The "Send" button that can be enabled in the module is not doing anything.
The button appears in relevant windows but when clicking on it, nothing appears in the chat.

This error is generated in the console:
![image](https://github.com/Dorako/pf2e-dorako-ux/assets/30345746/12c4620c-c07b-4b4c-b038-e81eb16858f0)

Issue persists with all other modules disabled.

Foundry version: 11.315
PF2E version: 5.13.6
Dorako UX version: Latest (1.3.2)

**Comment by JustADataConstruct** [comment]
Same problem here with same console log on Foundry 11.315, PF2e 5.16.1, and Dorako UX 1.3.6

**Comment by Dorako** [comment]
Yeah, this feature is in low maintenance mode. It doesn't work for all things, but it works for enough things that I'm keeping it around for now.

**Comment by farling42** [comment]
This can be fixed by adding parentheses around the new of the constructor call) in customization.js:

```js
await (new sheet.document.constructor(json, { parent: actor })).toChat();
```

(otherwise the code is trying to perform new on the result of the `toChat()` call.)
